games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="The FlightGear Flight Simulator"
```

Comment:
`SUMMARY` is not supposed to mention the name of the program. So maybe:
```bash
SUMMARY="A sophisticated flight simulator used by many professionals"
```
or
```bash
SUMMARY="A professional flight simulator used worldwide"
```
Hmm, I'm not convinced by my proposals...
Does anyone have another suggestion?

If there are no suggestion then I'm ok with keeping what you have right now. We can probably make an exception here and keep your
```bash
SUMMARY="The FlightGear Flight Simulator"
```